dbms/tests/performance/cpu_synthetic.xml

```diff
@@ -85,13 +83,11 @@ PageCharset тоже почти всегда непуст, но его сред
 <query>SELECT uniqCombined(17)(UserID) FROM hits_10m_single SETTINGS max_threads = 1</query>
 <query>SELECT uniqCombined(17)(UserID) FROM hits_100m_single</query>
 <!-- 44. Разные алгоритмы вычисления кардинальности. -->
-<query>SELECT uniqExact(UserID) FROM hits_10m_single SETTINGS max_threads = 1</query>
-<query>SELECT uniqExact(UserID) FROM hits_100m_single</query>
+<query>SELECT uniqExact(UserID) FROM hits_10m_single</query>
 <!-- 45. Что-то чуть более похожее на реальный запрос. -->
 <query>SELECT RegionID, uniq(UserID) FROM hits_10m_single GROUP BY RegionID SETTINGS max_threads = 1</query>
 <query>SELECT RegionID, uniq(UserID) FROM hits_100m_single GROUP BY RegionID</query>
 <!-- 47. Читаем и разжимаем все столбцы, и ничего с ними потом не делаем. -->
-<query>SELECT count() FROM hits_10m_single WHERE NOT ignore(*) SETTINGS max_threads = 1</query>
-<query>SELECT count() FROM hits_100m_single WHERE NOT ignore(*)</query>
+<query>SELECT count() FROM hits_10m_single WHERE NOT ignore(*)</query>
 
 </test>
```

Comment:
The purpose of CPU synthetic benchmark is not to test ClickHouse but to test CPU.
It's needed to validate the performance of a single CPU core. So, tests with max_threads = 1 must remain.